nautobot/docs/release-notes/version-2.2.md

```diff
@@ -44,3 +44,121 @@ The CSV import functionality for all models has been changed from a synchronous
 
 !!! tip
     Users now must have the `run` action permission for `extras > job` (specifically the `nautobot.core.jobs.ImportObjects` Job) in order to import objects, in addition to the normal `add` permissions for the object type being imported.
+
+<!-- towncrier release notes start -->
+## v2.2.0-beta.1 (2024-03-19)
+
+### Added
+
+- [#1](https://github.com/nautobot/nautobot/issues/1) - Added new models for software versions and software image files.
+- [#1](https://github.com/nautobot/nautobot/issues/1) - Added a many-to-many relationship from `Device` to `SoftwareImageFile`.
+- [#1](https://github.com/nautobot/nautobot/issues/1) - Added a many-to-many relationship from `DeviceType` to `SoftwareImageFile`.
+- [#1](https://github.com/nautobot/nautobot/issues/1) - Added a many-to-many relationship from `InventoryItem` to `SoftwareImageFile`.
+- [#1](https://github.com/nautobot/nautobot/issues/1) - Added a many-to-many relationship from `VirtualMachine` to `SoftwareImageFile`.
+- [#1](https://github.com/nautobot/nautobot/issues/1) - Added a foreign key relationship from `Device` to `SoftwareVersion`.
+- [#1](https://github.com/nautobot/nautobot/issues/1) - Added a foreign key relationship from `InventoryItem` to `SoftwareVersion`.
+- [#1](https://github.com/nautobot/nautobot/issues/1) - Added a foreign key relationship from `VirtualMachine` to `SoftwareVersion`.
+- [#230](https://github.com/nautobot/nautobot/issues/230) - Added Contact and Team Models.
+- [#1150](https://github.com/nautobot/nautobot/issues/1150) - Added environment variable support for most admin-configurable settings (`ALLOW_REQUEST_PROFILING`, `BANNER_TOP`, etc.)
+- [#3111](https://github.com/nautobot/nautobot/issues/3111) - Initial work on the controller model.
```

Comment:
Needs an entry in the release overview (part of #5454)

Reply:
Is that blocking this release?

Reply:
No I think we are going to continue iterating on the model after the release.